docs/developer-guide/nodejs-api.md

```diff
@@ -4,6 +4,35 @@ While ESLint is designed to be run on the command line, it's possible to use ESL
 
 **Note:** Use undocumented parts of the API at your own risk. Only those parts that are specifically mentioned in this document are approved for use and will remain stable and reliable. Anything left undocumented is unstable and may change or be removed at any point.
 
+## Table of Contents
+
+- [SourceCode](#sourcecode)
+  - [splitLines()](#sourcecodesplitlines)
+- [Linter](#linter)
+  - [verify()](#linterverify)
+  - [verifyAndFix()](#linterverifyandfix)
+  - [defineRule()](#linterdefinerule)
+  - [defineRules()](#linterdefinerules)
+  - [getRules()](#lintergetrules)
+  - [defineParser()](#linterdefineparser)
+  - [version()](#linterversion)
+- [linter (deprecated)](#linter-1)
+- [CLIEngine](#cliengine)
+  - [executeOnFiles()](#cliengineexecuteonfiles)
+  - [resolveFileGlobPatterns()](#cliengineresolvefileglobpatterns)
+  - [resolveFileGlobPatterns](#cliengineresolveFileGlobPatterns)
```

Comment:
This line appears to be a near-duplicate of the previous line. I think this line can be removed (and the previous line preserved).